app/serializers/move_serializer.rb

```diff
@@ -4,13 +4,13 @@ class MoveSerializer < ActiveModel::Serializer
   attributes :id, :reference, :status, :updated_at, :time_due, :date, :move_type, :additional_information,
              :cancellation_reason, :cancellation_reason_comment
 
-  has_one :person, serializer: PersonSerializer
+  has_one :profile, serializer: ProfileSerializer
   has_one :from_location, serializer: LocationSerializer
   has_one :to_location, serializer: LocationSerializer
   has_many :documents, serializer: DocumentSerializer
 
   INCLUDED_ATTRIBUTES = {
-    person: %i[first_names last_name date_of_birth assessment_answers indentifiers ethnicity gender],
+    profile: %i[first_names last_name date_of_birth assessment_answers indentifiers ethnicity gender],
```

Comment:
This looks like it removes an attribute the API response, which we're not able to do. The API is closed for modification, but open for extension. As galling as it is to introduce inconsistency, we may need to keep the `person` name here.

Reply:
pretty sure it doesn't do that. There are no test changes in this PR. The new 'ProfileSerializer' class has a type 'people' line to keep the API consistent